brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-610 Improvement: Remove word "device" from the tooltip in responsive icons.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/b1c2d232-e4f3-418a-b107-2207d0ac04fb?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/NQuNQkLB

### Types of changes




- Improvement

### How has this been tested?

- Click on the Responsive Icon in the editor and check the tooltip.
- Check other component like Responsive slider, Spacing and wherever we added responsive support.
- There should be device name on tooltip, no static string.

### Checklist:
